src/test/java/uk/gov/justice/digital/converter/dms/DMS_3_4_6Test.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.converter.dms;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DMS_3_4_6Test {
+
+    private static SparkSession sparkSession;
+
+    @BeforeAll
+    public static void beforeAll(){
+        sparkSession = SparkSession.builder().master("local[*]").getOrCreate();
+    }
+
+    @ParameterizedTest
```

Comment:
I don't think using a parameterized test here is buying us much when we can just have the string declared in the method for now.